### DISCUSSION TITLE
Github copilot 404 cannot install - Jetbrains IntelliJ

### DISCUSSION THREAD

**Discussion by thejeff77** [community/community#7732] [Copilot Conversations] [answered]
Anyone else getting a complete failure to install it?

Thanks for any help - see screenshot

![Screen Shot 2021-11-15 at 6 53 29 AM](https://user-images.githubusercontent.com/8584457/141793551-04d5056e-96fe-4f0a-bc6d-1506a9641f66.png)

**Comment by jansorg** [comment, marked as answer]
That's weird. The downloaded files are served by JetBrains.
If restarting the IDE doesn't help, you could try to download the latest release at https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/144352 and install it manually, e.g. by Drag&Drop onto the main window of your IDE.

**Comment by thejeff77** [comment]
Yeah restarting didn't help. After dragging the jar files in, then checking plugins - it still shows as not installed. Any other ideas?

**Reply by jansorg** [reply]
Drag&Drop the zip file, not the jar files contained in it.
AFAIK JetBrains is aware of the 404 issue, it will hopefully be solved soon.

**Comment by raksha102** [comment]
Make sure you have logged in to GitHub Account. Or else it fails to install plugin. Even I was getting the same error. Finally fixed by logging in.

**Reply by rcardin** [reply]
Thanks, man. You saved my day!

**Reply by mosab7** [reply]
Thanks, works for me too.
